static/admin/admin.js

```diff
@@ -152,6 +152,54 @@ var GuestPreview = createClass({
 })
 CMS.registerPreviewTemplate("guests", GuestPreview)
 
+var HotelPreview = createClass({
+    render: function() {
+        const entry = this.props.entry.toJS()
+        const data = entry.data
+        const bookingStatus = data.bookingStatus
+        const isOpen = bookingStatus === 'open'
+        const isClosed = bookingStatus === 'closed'
+        if (!isOpen) {
+            let message = "Online hotel booking is now closed. Please [contact us] if you would like assistance booking a room at our discounted rates."
+            if (!isClosed) {
```

Comment:
This is a bit of a nitpick but can we please use an `isWaiting` variable in both places instead of this? It's not super clear what this is doing during a quick read, and if we ever added another state this could lead to weird behavior.